scripts/analysisTools/plotUtils/utility.py

```diff
@@ -3684,19 +3692,27 @@ def roll1Dto2D(h1d, histo, invertXY=False):
 
     return histo
 
-def unroll2Dto1D(h, newname='', cropNegativeBins=True, silent=False):
+def unroll2Dto1D(h, newname='', cropNegativeBins=True, silent=False, invertUnroll=False):
```

Comment:
Can't you just do the unrolling before converting to ROOT? The boost hist scripts for this will be much faster

Reply:
Yes, but this function is typically used in places where you already have root files, and was actually inherited from the code before boost started to be used.

Reply:
Sure, I'm not saying I would delete the function, but keep in mind that when you can use the boost ones, it's better